satellite/console/consoleweb/server.go

```diff
@@ -103,13 +108,17 @@ func (s *Server) grapqlHandler(w http.ResponseWriter, req *http.Request) {
 		ctx = console.WithAuth(ctx, auth)
 	}
 
+	rootObject := make(map[string]interface{})
+	rootObject["origin"] = "http://" + s.config.Address + "/"
```

Comment:
Note `s.config.Address` might be `127.0.0.1:0` here. Also since it's a public name you might also need to specify host (for the production) rather than the address.